### PR TITLE
Typo fixes for doxygen and source comments

### PR DESCRIPTION
**Pull request by luzpaz**
issue #0002914

### CODE REVIEW

File: src/Tools/SubWCRev.py [hunk 1]
```diff
@@ -170,12 +170,12 @@ def geturl(self):
...
-the first number reflects the number of commits in common with the
-blessed master repository.
-the second part, seperated by " +"reflects the number of commits that are
-different form the master repository"""
+        for master and release branches all commits are counted
+        for other branches the version numver is split in two parts
+        the first number reflects the number of commits in common with the
+        blessed master repository.
+        the second part, separated by " +" reflects the number of commits that are
+        different frm the master repository"""
```
Comment (luzpaz) [inline src/Tools/SubWCRev.py]: need to fix that typo i introduced. Will repush after I do so.

File: src/App/Enumeration.h [hunk 2]
```diff
@@ -148,13 +148,13 @@ namespace App
             /// Assignment operator
             Enumeration & operator=(const Enumeration &other);
 
-            /// true iff our string representation matches other's
+            /// true if our string representation matches others
```
Comment (ianrrees) [inline src/App/Enumeration.h]: That's intentional...

Comment (luzpaz) [inline src/App/Enumeration.h]: `iff` or `other's`

Comment (ianrrees) [inline src/App/Enumeration.h]: I've explained to you, at some length, that I think these comment edits are a waste of time.  So, sorry, but I'm not going to put further effort in this direction.

**Comment by yorikvanhavre** [comment]
merged

**Pull request closed without merging**